### PR TITLE
Added mention of updated Boostrap

### PR DESCRIPTION
**Pull request by mnocon**
Follow-up to https://github.com/ibexa/documentation-developer/pull/2325#pullrequestreview-1935903597 , after Vidar's comment

### CODE REVIEW

File: docs/update_and_migration/from_3.3/to_4.0.md [hunk 1]
```diff
@@ -161,6 +161,10 @@ php bin/console ibexa:migrations:migrate
 
 ## Update your custom code
 
+### Back Office customizations
+
+The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you've used Bootstrap 4 to style your custom Back Office components, rewrite them [using the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).
```
Comment (adamwojs) [inline docs/update_and_migration/from_3.3/to_4.0.md:166]: IMHO "migrate to" / "upgrade to" sound better then "rewrite"

**Review by dew326 — APPROVED** [review]

File: docs/update_and_migration/from_3.3/to_4.0.md [hunk 2]
```diff
@@ -161,6 +161,10 @@ php bin/console ibexa:migrations:migrate
 
 ## Update your custom code
 
+### Back Office customizations
+
+The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you've used Bootstrap 4 to style your custom Back Office components, update them [using the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_3.3/to_4.0.md:166]:
```suggestion
The v4 version of [[= product_name =]] is using Bootstrap 5 in the Back Office. If you were using Bootstrap 4 for styling, you need to update and adjust all custom Back Office components [following the migration guide from Bootstrap 4](https://getbootstrap.com/docs/5.0/migration/).
```

File: docs/update_and_migration/from_3.3/to_4.0.md [hunk 3]
```diff
@@ -161,6 +161,10 @@ php bin/console ibexa:migrations:migrate
 
 ## Update your custom code
 
+### Back Office customizations
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_3.3/to_4.0.md:164]:
```suggestion
### Back Office customization
```

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
Please check my comments.

**Review by adamwojs — APPROVED** [review]

**Review by dabrt — APPROVED** [review]

**Pull request merged**